mmv1/third_party/terraform/services/resourcemanager/resource_google_project_service_test.go

```diff
@@ -323,3 +358,20 @@ resource "google_project_service" "test" {
 }
 `, pid, pid, org, service)
 }
+
+func testAccProjectService_checkUsage(service string, pid, org string, checkIfServiceHasUsage string) string {
+	return fmt.Sprintf(`
+resource "google_project" "acceptance" {
+  project_id = "%s"
+  name       = "%s"
+  org_id     = "%s"
+}
+
+resource "google_project_service" "test" {
+  project = google_project.acceptance.project_id
+  service = "%s"
+
+  check_if_service_has_usage_on_destroy = checkIfServiceHasUsage
+}
+`, pid, pid, org, service)
```

Comment:
```suggestion
func testAccProjectService_checkUsage(service string, pid, org string, checkIfServiceHasUsage string) string {
	return fmt.Sprintf(`
resource "google_project" "acceptance" {
  project_id = "%s"
  name       = "%s"
  org_id     = "%s"
}

resource "google_project_service" "test" {
  project = google_project.acceptance.project_id
  service = "%s"

  check_if_service_has_usage_on_destroy = %s
}
`, pid, pid, org, service, checkIfServiceHasUsage)
```

this needs to use formatted string syntax to get in your new argument